rayon-core/src/lib.rs

```diff
@@ -144,6 +144,13 @@ impl Configuration {
         if self.num_threads > 0 {
             self.num_threads
         } else {
+            match env::var("RAYON_RS_NUM_THREADS").ok().and_then(|s| usize::from_str(&s).ok()) {
+                Some(x) if x > 0 => return x,
+                Some(x) if x == 0 => return num_cpus::get(),
+                _ => {},
+            }
+
+            // Support for deprecated `RAYON_RS_NUM_CPUS`.
             match env::var("RAYON_RS_NUM_CPUS").ok().and_then(|s| usize::from_str(&s).ok()) {
```

Comment:
Meh, I'd probably just leave it. However, to be strictly backwards compatible, shouldn't we prefer `RAYON_RS_NUM_CPUS` if both are given?